doc/getting-started.md

```diff
@@ -1,11 +1,17 @@
 # Getting started with Istio
```

Comment:
I think this tutorial should not be named "gettting-started.md" (i.e., the target of the "getting started" link in the main istio README.md). IMO, the getting started page should be some setup instructions and then a list of links to tutorials users can try, to learn various things, this tutorial being one of them. The title of this one should be something more like "Understanding Istio proxy injection in Kubernetes".